core/src/main/java/com/cyberbot/bomberman/core/models/tiles/loader/TileMapFactory.java

```diff
@@ -0,0 +1,120 @@
+package com.cyberbot.bomberman.core.models.tiles.loader;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.physics.box2d.World;
+import com.cyberbot.bomberman.core.models.tiles.*;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.File;
+import java.io.IOException;
+import java.util.InvalidPropertiesFormatException;
+
+public class TileMapFactory {
+    private static final String TAG = TileMapFactory.class.getSimpleName();
+
+    private static final String LAYER_BASE = "base";
+    private static final String LAYER_FLOOR = "floor";
+    private static final String LAYER_WALLS = "walls";
+
+
+    public static TileMap createTileMap(World world, String path) throws IOException, MissingLayersException, ParserConfigurationException, SAXException {
+
+        Node sourceMap = loadXmlMap(path);
+        Element mapElement = (Element) sourceMap;
+        NodeList tilesetSource = mapElement.getElementsByTagName("tileset");
+        Element tilesetSourceElement = (Element) tilesetSource.item(0);
+
+        NodeList tileset = loadXmlTileset("./map/" + tilesetSourceElement.getAttribute("source"));
```

Comment:
The path here should be handled better, ex. get the folder from the `path` variable (use `File`)

Reply:
Fixed